aws-opentelemetry-distro/src/amazon/opentelemetry/distro/aws_opentelemetry_configurator.py

```diff
@@ -85,6 +86,8 @@ class AwsOpenTelemetryConfigurator(_OTelSDKConfigurator):
     # pylint: disable=no-self-use
     @override
     def _configure(self, **kwargs):
+        if _is_defer_to_workers_enabled() and _is_wsgi_master_process():
+            return
```

Comment:
Can we add a `info` level log message when skipping?

Reply:
And `pid` in the log message as well?

Reply:
Good idea. will add that rn.